src/main/java/org/venice/beachfront/bfapi/BfApiConfig.java

```diff
@@ -105,12 +109,17 @@
 import springfox.documentation.service.Contact;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
+import util.GeoAxisJWTUtility;
 import util.PiazzaLogger;
 
 @Configuration
 public class BfApiConfig {
 	@Value("${http.keep.alive.duration.seconds}")
 	private int httpKeepAliveDurationSeconds;
+	@Value("${GEOAXIS_JWT_CERT:}")
+	private String geoaxisJwtCertPath;
```

Comment:
This isn't really the path to the cert, but rather the contents of the cert.

Reply:
We load using  `getClass().getClassLoader().getResourceAsStream(geoaxisJwtCertPath))` , so it is the resource path.  Since we put the file into src/main/resources, we just access it via the filename directly.